app/views/shared/menu/_large_menu.html.haml

```diff
@@ -37,6 +37,12 @@
               = t 'label_about'
 
     %ul.right
+      %li.powered-by
+        %img{src: '/favicon.ico'}
+        %span
+          = t 'powered_by'
+          %a{href: '/'}
```

Comment:
I haven't tested this yet. Is the homepage allowed to be embedded? Or should we open a new window here? Would we have a problem with the embedded state if people browse full OFN and then go back to the embedded tab?